vc2017x86run.txt

```diff
@@ -6,6 +6,7 @@ Description = Needed by applications compiled with Visual Studio 2017.
 Size = 14,56 MiB
 Category = 14
 URLSite = https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads
+URLSite1 = https://github.com/WPN-XM/vcredist
```

Comment:
What that is for? There is nothing but a readme at this URL.

Reply:
the possible for add 1,2 other url's if give a Webspace , github and maybe a Forumspage .
therefore should we have a possible for ulrside ,1,2,3.

Also would it be great if we have a "unattended=" or "swiches =" possible
best regards
Blacky